packages/fiori-elements-writer/test/common.ts

```diff
@@ -0,0 +1,120 @@
+import { join } from 'path';
+import { readFileSync } from 'fs';
+import { create as createStore } from 'mem-fs';
+import { create } from 'mem-fs-editor';
+import {
+    FEOPSettings,
+    FioriElementsApp,
+    LROPSettings,
+    OVPSettings,
+    Template,
+    TemplateType,
+    WorklistSettings
+} from '../src/types';
+import { OdataService, OdataVersion } from '@sap-ux/odata-service-writer';
+
+export const testOutputDir = join(__dirname, '/test-output');
```

Comment:
Maybe you don't need the ```/``` in ```/test-output```. Win support?

Reply:
Yes not needed will remove...but join would convert for win anyway.